examples/jsm/nodes/core/StackNode.js

```diff
@@ -68,6 +69,23 @@ class StackNode extends Node {
 
 	assign( targetNode, sourceValue ) {
 
+		sourceValue = nodeObject( sourceValue );
```

Comment:
I don't think we need all of this, we can just remove this function and rely on ShaderNode.

Reply:
(Fixed in https://github.com/mrdoob/three.js/pull/26821)